auditeval/test.go

```diff
@@ -70,45 +76,58 @@ func (t *testItem) execute(s string) (result bool) {
 				os.Exit(1)
 			}
 
+			result.ActualResult = strings.ToLower(flagVal)
 			switch t.Compare.Op {
 			case "eq":
-				result = flagVal == t.Compare.Value
+				value := strings.ToLower(flagVal)
+
+				// Do case insensitive comparaison for booleans ...
```

Comment:
Good job finding this in the kube-bench code! (Though I guess it just goes to show, we really need to finish the job of using the common code from both the docker & kube benches)